### DISCUSSION TITLE
Added service renovate user, no MR is ever created.

### DISCUSSION THREAD

**Discussion by LucasMali** [renovatebot/renovate#19257] [Request Help] [answered]
Hello,

Our team is using a platform that has renovate ready to go. The configuration json file is in the root directory, the service-renovate user has been added as well. However, we never see an onboarding MR triggered from renovate. What are some ways we can troubleshoot this?

Best,
L

**Comment by HonkingGoose** [comment]
Thanks for trying out Renovate! We have [docs about troubleshooting](https://docs.renovatebot.com/troubleshooting/), that's probably a good point to start.

In any case, we need to know:

1. How long has it been since you ran/enabled Renovate bot for the first time?
1. What platform are you running Renovate on GitLab, GitHub, something else?
1. Are you self-hosting Renovate or using the GitHub-hosted app of Renovate?
   - If you're self-hosting, what version of Renovate are you using?
1. Are there any debug messages or warnings?

We also need your Renovate configuration. Remember to remove any private or secret information before posting it. 😉

**Comment by LucasMali** [comment, marked as answer]
It was settings in our repository, allowed branch names and MR required approvals. Adding in renovate to the allowed branch names and giving the user bypass MR approval, it works. Thanks.
